### DISCUSSION TITLE
Don’t show the error overlay

### DISCUSSION THREAD

**Discussion by KaiseiSato** [gatsbyjs/gatsby#30973] [Help] [answered]
## Description

If a runtime error occurs, the error overlay will be forced to appear and hot reloading will stop working.
But I am aware and allow that the runtime error occurs.
So I want to ignore the runtime error or I don't want to show the error overlay.

### Steps to reproduce

Cause a runtime error.

### Expected result

Error overlay do not appear.

### Actual result

Error overlay is forced to appear

![screenshot](https://user-images.githubusercontent.com/50479525/115497991-a9fb7800-a2a7-11eb-9d5a-308c029e1ab3.png)

### Environment

System:
    OS: macOS 11.2.3
    CPU: (8) x64 Intel(R) Core(TM) i7-8569U CPU @ 2.80GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.16.2 - ~/.nodebrew/current/bin/node
    Yarn: 1.19.1 - /usr/local/bin/yarn
    npm: 6.9.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 89.0.4389.128
    Firefox: 84.0.2
    Safari: 14.0.3
  npmPackages:
    gatsby: ^3.2.1 => 3.2.1
    gatsby-plugin-eslint: ^2.0.8 => 2.0.8
    gatsby-plugin-react-helmet: ^4.2.0 => 4.2.0
    gatsby-plugin-react-helmet-canonical-urls: ^1.4.0 => 1.4.0
    gatsby-plugin-root-import: ^2.0.5 => 2.0.5
    gatsby-plugin-stylus: ^3.2.0 => 3.2.0
    gatsby-plugin-typegen: ^2.2.4 => 2.2.4
    gatsby-source-microcms: ^1.0.1 => 1.0.1

**Comment by LekoArts** [comment, marked as answer]
You can press the "X"-Button to hide the overlay when it occurs but we won't provide a way to completely disable it.
